src/pages/MessagesPage.tsx

```diff
@@ -34,59 +35,63 @@ const MessagesPage = () => {
         }}
       >
         <Messages onClick={handleClick} />
-        <Box>
-          {selectedChat && (
-            <Box className={classes.header}>
-              <Box className={classes.userInHeader}>
-                <Avatar
-                  src={selectedChat.avatar}
-                  sx={{ width: 50, height: 50 }}
-                />
-                <Typography
-                  sx={{
-                    color: theme.palette.primary.main,
-                    fontWeight: 600,
-                    fontSize: 20,
-                  }}
-                >
-                  {selectedChat.firstName} {selectedChat.lastName},{' '}
-                  {selectedChat.age}
-                </Typography>
-              </Box>
+        {selectedChat ? (
+          <Box>
+            {selectedChat && (
```

Comment:
We already have the condition "selectedChat ?" above, so I guess here "selectedChat &&" is useless? because it will be always true, right?

Reply:
Removed
You are right, I don't need this condition. It's left from the previous version.